tokio-postgres/src/connect_raw.rs

```diff
@@ -141,8 +141,13 @@ where
     T: AsyncRead + AsyncWrite + Unpin,
 {
     match stream.try_next().await.map_err(Error::io)? {
```

Comment:
Should we have a comment here to clarify that this is only dealing with the first message in the authentication sequence? I was slightly confused at first, thinking that this would reject any ``AuthenticationOk`` message if ``channel_binding=require``, but it only rejects it if it's the first in the sequence.